modules/nested-tabs/assets/scss/frontend.scss

```diff
@@ -169,7 +162,7 @@
 				}
 			}
 
-			&:not( .e-active ):hover {
+			&:not( .e-active ).e-normal:hover {
```

Comment:
I have removed the hover styling on the mobile version.
The combination of the hover styling and the active styling caused a glitch visually.